src/w3c/headers.js

```diff
@@ -429,15 +429,15 @@ export function run(conf) {
         alt.hasOwnProperty("lang") && alt.lang ? ` hreflang='${alt.lang}'` : "";
       optional +=
         alt.hasOwnProperty("type") && alt.type ? ` type='${alt.type}'` : "";
-      return `<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
+      return hyperHTML `<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
```

Comment:
We need a new `joinAnd`-like function but for `HTMLElement`s instead of strings.

Reply:
Should i write a new joinAnd functioin or change the existing one. If a new has to be written, where should I write it

Reply:
The new one should go `src/core/utils.js` 👍